rcp/org.testeditor.rcp4.views.projectexplorer/src/org/testeditor/rcp4/views/projectexplorer/TEContentProvider.xtend

```diff
@@ -42,11 +46,25 @@ class TEContentProvider implements ITreeContentProvider {
 	}
 
 	override getParent(Object element) {
+		if (element instanceof IResource) {
+			if (element.project != null) {
+				val list = getChildren(element.project).filter(IClasspathEntry).filter [
```

Comment:
Can we extract this to a helper method? As I understand it, it retrieves the first `IClasspathEntry` the contains the passed element, doesn't it?